src/k8s/api/v1/cluster_config.go

```diff
@@ -13,7 +13,11 @@ type GetClusterConfigResponse struct {
 }
 
 type UpdateClusterConfigRequest struct {
-	Config UserFacingClusterConfig
+	Config UserFacingClusterConfig `json:"config,omitempty" yaml:"config,omitempty"`
```

Comment:
With fields getting added here, would it make any sense to update the name of `UserFacingClusterConfig` to something like `UserFacingClusterFeatureConfig`?